packages/api/src/routes/taskRoute.js

```diff
@@ -59,14 +62,17 @@ router.post('/scouting_task', modelMapping['scouting_task'],
 router.post('/soil_task', modelMapping['soil_task'],
   hasFarmAccess({ body: 'locations' }), isWorkerToSelfOrAdmin, taskController.createTask('soil_task'));
 
+router.post('/custom/other_task', modelMapping['soil_task'],
+  hasFarmAccess({ body: 'locations' }), isWorkerToSelfOrAdmin, taskController.createTask('soil_task'));
+
 router.post('/field_work_task', modelMapping['field_work_task'],
   hasFarmAccess({ body: 'locations' }), isWorkerToSelfOrAdmin, taskController.createTask('field_work_task'));
 
 router.post('/harvest_task', modelMapping['harvest_task'],
   hasFarmAccess({ body: 'locations' }), isWorkerToSelfOrAdmin, taskController.createTask('harvest_task'));
 
-router.post('/plant_task', modelMapping['plant_task'],
-  hasFarmAccess({ body: 'locations' }), isWorkerToSelfOrAdmin, taskController.createTask('plant_task'));
+router.post('/transplant_task', modelMapping['transplant_task'],
+  hasFarmAccess({ mix: 'transplant_task' }), isWorkerToSelfOrAdmin, taskController.createTransplantTask());
```

Comment:
will there be no create plant task route? i assume this is because plat task is only created upon management plan creation right?